src/measure_theory/measure_space.lean

```diff
@@ -377,6 +377,25 @@ begin
   exact tendsto_at_top_infi_nat (μ ∘ s) (assume n m hnm, measure_mono $ hm _ _ $ hnm),
 end
 
+/-- The Borel-Cantelli lemma. -/
```

Comment:
Can you expand the docstring, especially since there are two Borel-Cantelli lemmas and the one here is the easy one.

Also, could you state the version saying that, for almost every `x`, eventually, `x` does not belong to `s n`?